src/mem3_rpc.erl

```diff
@@ -275,6 +275,16 @@ rexi_call(Node, MFA) ->
     end.
 
 
+get_or_create_db(DbName, Options) ->
```

Comment:
since this is a cut-and-paste of a live function in fabric, can we put some thought to keeping this DRY?